overtime_analyzer.py

```
@@ -0,0 +1,941 @@
+                    )  # 초과근무시간 정보 (있는 경우 사용)
+                    overtime_hours = None
+                    if "초과근무시간" in df.columns and pd.notna(row["초과근무시간"]):
+                        try:
+                            # 문자열이면 숫자로 변환 시도
+                            if isinstance(row["초과근무시간"], str):
+                                # 콤마, 공백 등 제거하고 숫자 변환
+                                clean_str = row["초과근무시간"].replace(",", "").strip()
+                                overtime_hours = float(clean_str)
+                            else:
+                                overtime_hours = float(row["초과근무시간"])
+                        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)